.github/workflows/python-ci.yml

```
@@ -0,0 +1,31 @@
+name: Python CI
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
```

```suggestion
  push:
    branches: [ main ]
    paths: [ deploy/grafana-dashboard ]
  pull_request:
    branches: [ main ]
    paths: [ deploy/grafana-dashboard ]
```

We could add a filter so it does not run unnecessarily

---

Sure, that's a great addition. Thanks!